### PR TITLE
Implemented top speaker service

### LINKED ISSUE #5: Top speaker service
### Need to implement the following services in the top speaker service
- Add speaker
- Get all speaker
- Update speaker
- Delete speaker

### PR DESCRIPTION
**Pull request by miyurugunarathna**
Closes #5

### CODE REVIEW

File: src/api/services/TopSpeaker.service.ts [hunk 1]
```diff
@@ -1,23 +1,91 @@
+import { DocumentDefinition } from "mongoose";
+import { ITopSpeaker } from "../interfaces";
+import TopSpeakerModel from "../models/TopSpeaker.model";
 /**
  * @todo create @function insertTopSpeaker to save a speaker in the database
```
Comment (rusiruavb) [inline src/api/services/TopSpeaker.service.ts:5]: In the next time onwards, remove the `todo` item from the comment and add a proper comment for the function.

**Review by rusiruavb — APPROVED** [review]
LGTM

**Comment by rusiruavb** [comment]
@miyurugunarathna are the controller functions implemented for this service?

File: src/api/services/TopSpeaker.service.ts (hunk 1 above)
Comment (miyurugunarathna) [reply src/api/services/TopSpeaker.service.ts:5]: Okay. Got it.

**Comment by miyurugunarathna** [comment]
> @miyurugunarathna are the controller functions implemented for this service?

Not yet. @nisalrenuja will do it.

File: src/api/services/TopSpeaker.service.ts (hunk 1 above)
Comment (nisalrenuja) [reply src/api/services/TopSpeaker.service.ts:5]: I will implement the rest of the things in the top speaker service and controller part

> > @miyurugunarathna are the controller functions implemented for this service?
> 
> Not yet. @nisalrenuja will do it.

I will implement the rest of the things in the top speaker service and controller part

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**